ServiceBased/ProAddIn/WorkflowManagerSampleAddIn/Module1.cs

```diff
@@ -41,7 +41,31 @@ protected override bool CanUnload()
             return true;
         }
 
-        #endregion Overrides
+        /// <summary>
+        /// Override this method to allow execution of DAML commands specified in this module.
+        /// This is needed to run commands using the Open Pro Project Items step.
+        /// </summary>
+        /// <param name="id">The DAML control identifier.</param>
+        /// <returns>A user defined function that will execute asynchronously when invoked.</returns>
+        protected override Func<Task> ExecuteCommand(string id)
+        {
+            return () => QueuedTask.Run(() =>
+            {
+                try
+                {
+                    // Run the command specified by the id
+                    IPlugInWrapper wrapper = FrameworkApplication.GetPlugInWrapper(id);
```

Comment:
Do we want to look up any command like this, or should be delegating to specific commands defined in this module?

Reply:
@kbedel - If the command spacified by the id is not defined in this module, then this method will not be called.

Reply:
Is this how we'd want users to actually implement this though? Or would we expect more like what Editing does here: https://devtopia.esri.com/ArcGISPro/ArcGISPro/blob/main/ArcGISDesktop/ArcGISEditing/ArcGIS.Desktop.Editing/EditingModule.cs#L5805-L5806

Reply:
Chatted with @kbedel and this is fine for the current implementation. We'll update samples again when we make changes in Pro 3.4.